### PR TITLE
Form - simple layout manager implementation

### PR DESCRIPTION
**Pull request by novsstation**

### CODE REVIEW

File: js/renovation/ui/form/form_layout_manager.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <Widget
+      classes={cssClasses}
+      {...restAttributes} // eslint-disable-line react/jsx-props-no-spreading
+    />
+  );
+};
+
+@Component({
+  defaultOptionRules: null,
+  jQuery: { register: true },
```
Comment (EugeniyKiyashko) [inline js/renovation/ui/form/form_layout_manager.tsx]: Shall we register layout manager? It is not public component

Comment (novsstation) [reply js/renovation/ui/form/form_layout_manager.tsx]: Sorry, it's my fault

File: js/renovation/ui/form/__tests__/form_layout_manager.test.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { mount } from 'enzyme';
+import React from 'react';
```
Comment (IgnatovDan) [inline js/renovation/ui/form/__tests__/form_layout_manager.test.tsx:2]: Is 'React' used?

**Review by IgnatovDan — CHANGES_REQUESTED** [review]

File: js/renovation/ui/form/__tests__/form_layout_manager.test.tsx (hunk 2 above)
Comment (novsstation) [reply js/renovation/ui/form/__tests__/form_layout_manager.test.tsx:2]: Yep, otherwise there will be a warning
image

**Review by IgnatovDan — APPROVED** [review]

**Pull request merged**